### PR TITLE
added DAM in parabola topo, renamed/cleaned topo naming 

### PR DESCRIPTION
**Pull request by jtfisch**
- watch out if the change in ToPo Name changes have consequences somewhere else.
- also created an issue that the parameters should be ordered according to topos

### CODE REVIEW

File: avaframe/in3Utils/generateTopo.py [hunk 1]
```diff
@@ -104,11 +104,11 @@ def inclinedplane(cfg):
         # make half sphere shaped channel with radius given by channel horizontal extent
         mask = np.zeros(np.shape(yv))
         mask[np.where(abs(yv) < c_extent)] = 1
-        if cfg['TOPO'].getboolean('topoconst'):
-            zv = zv - c_extent*c_0*np.sqrt(np.abs(1. - (np.square(yv) / (c_extent**2))))*mask
-        else:
+        if cfg['TOPO'].getboolean('topoadd'):
```
Comment (fso42) [inline avaframe/in3Utils/generateTopo.py:107]: Please stick to our naming convention, i.e.
```suggestion
        if cfg['TOPO'].getboolean('topoAdd'):
```

Maybe this is an opportunity to finally change all the ones that are in the Ini file, i.e c_extent to cExtent and so on...

File: avaframe/in3Utils/generateTopo.py [hunk 2]
```diff
@@ -266,18 +270,35 @@ def hockey(cfg):
...
             # outside of the channel, add layer of channel depth
             mask = np.zeros(np.shape(y))
             mask[np.where(abs(y) >= c_extent)] = 1
-            zv = zv + c_extent*c_0*mask
+            #zv = zv + c_extent*c_0*mask
+            superChannel = superChannel + c_extent*c_0*mask
+            
+    if cfg['TOPO'].getboolean('dam'):
+        #damPos = float(cfg['DAMS']['damPos'])
```
Comment (fso42) [inline avaframe/in3Utils/generateTopo.py:287]: please delete

File: avaframe/in3Utils/generateTopo.py [hunk 3]
```diff
@@ -266,18 +270,35 @@ def hockey(cfg):
...
+        damPos = cfg['DAMS'].getfloat('damPos')
+        damheight = cfg['DAMS'].getfloat('damheight')
```
Comment (fso42) [inline avaframe/in3Utils/generateTopo.py:289]:
```suggestion
        damHeight = cfg['DAMS'].getfloat('damHeight')
```

File: avaframe/in3Utils/generateTopo.py [hunk 4]
```diff
@@ -266,18 +270,35 @@ def hockey(cfg):
...
+        damwidth = cfg['DAMS'].getfloat('damwidth')
```
Comment (fso42) [inline avaframe/in3Utils/generateTopo.py:290]:
```suggestion
        damWidth = cfg['DAMS'].getfloat('damWidth')
```

File: avaframe/in3Utils/generateTopoCfg.ini [hunk 5]
```diff
@@ -89,14 +91,22 @@ c_muendFP = 0.86
...
 #--------------------------------------------------------------
 
+# DAM parameters -----------------------------------
+[DAMS]
+# relative position with respect to where flat outrun starts
+damPos = 0.6 
+
+damwidth = 75
+
+damheight = 75
```
Comment (fso42) [inline avaframe/in3Utils/generateTopoCfg.ini:101]: Naming convention, see above

File: avaframe/in3Utils/generateTopoCfg.ini [hunk 6]
```diff
@@ -89,14 +91,22 @@ c_muendFP = 0.86
...
 
 #DEM outputfile parameters---------------------------------
 [DEMDATA]
 # x coordinate of lower left corner
-xl = 1000.0
+xl = 0.0
 
 # y-coordinate of lower left corner
-yl = -5000.0
+yl = 0.0
```
Comment (fso42) [inline avaframe/in3Utils/generateTopoCfg.ini:109]: Should this be changed?

**Review by fso42 — CHANGES_REQUESTED** [review]

**Comment by fso42** [comment]
Oh, one thing I forgot: please adjust the test_generateTopo to account for your changes -> pytest fails at the moment...

**Review by matthiasto — APPROVED** [review]

**Pull request merged**